__test__/component/form/CreateCustomerForm.test.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
```

Comment:
Shouldn't the test folder be named __tests__?

Reply:
Sure, let's call it tests